src/app/health/health-event.component.ts

```diff
@@ -46,6 +46,17 @@ export class HealthEventComponent implements OnInit {
     });
   }
 
+  isEmptyForm() {
+    return this.healthForm.controls.temperature.value === '' && this.healthForm.controls.pulse.value === '' &&
```

Comment:
@ScottHuangNYU Please change to loop over the controls rather than explicitly checking each field.  This would make it more difficult to maintain if we add or remove form fields.

Reply:
@paulbert I changed to loop over the controls. But I wonder why I tried to return false in the ` (this.healthForm.controls[key].value !== '') ` if statement instead of using a `notAllEmpty` to make the `isEmptyForm` function to return false after I made any input can not work?